src/Raven.Studio/typescript/viewmodels/common/timeSeriesPlotDetails.ts

```diff
@@ -302,6 +305,27 @@ class timeSeriesPlotDetails extends viewModelBase {
             }
         });
         
+        this.tabText = ko.pureComputed(() => {
+           const documentIdsSet = new Set<string>();
+           const allData = [...this.rangeTimeSeries, ...this.pointTimeSeries];
+           allData.map(x => documentIdsSet.add(genUtils.escapeHtml(x.documentId)));
+           
+           const numberOfDocuments =  Array.from(documentIdsSet).length;
+           return numberOfDocuments === 1 ? `TS - ${genUtils.truncateDocumentId(allData[0].documentId)}` :
+                                            `TS - ${numberOfDocuments} documents`;
+        });
+        
+        this.tabInfo = ko.pureComputed(() => {
+           const documentIdsSet = new Set<string>();
+           const allData = [...this.rangeTimeSeries, ...this.pointTimeSeries];
+           allData.map(x => documentIdsSet.add(`<span class="document-id">${genUtils.escapeHtml(x.documentId)}</span> - ${x.name}`));
```

Comment:
can we avoid adding to set formatted values? Instead format then when creating html below. In case of duplicates there is less work to do + less memory

Reply:
done